test/system/030-run.bats

```diff
@@ -746,20 +746,23 @@ EOF
 }
 
 @test "podman run --device-cgroup-rule tests" {
-    skip_if_rootless "cannot add devices in rootless mode"
-
-    run_podman run --device-cgroup-rule="b 7:* rmw" --rm $IMAGE
-    run_podman run --device-cgroup-rule="c 7:* rmw" --rm $IMAGE
-    run_podman run --device-cgroup-rule="a 7:1 rmw" --rm $IMAGE
-    run_podman run --device-cgroup-rule="a 7 rmw" --rm $IMAGE
-    run_podman 125 run --device-cgroup-rule="b 7:* rmX" --rm $IMAGE
-    is "$output" "Error: invalid device access in device-access-add: X"
-    run_podman 125 run --device-cgroup-rule="b 7:2" --rm $IMAGE
-    is "$output" 'Error: invalid device cgroup rule requires type, major:Minor, and access rules: "b 7:2"'
-    run_podman 125 run --device-cgroup-rule="x 7:* rmw" --rm $IMAGE
-    is "$output" "Error: invalid device type in device-access-add: x"
-    run_podman 125 run --device-cgroup-rule="a a:* rmw" --rm $IMAGE
-    is "$output" "Error: strconv.ParseUint: parsing \"a\": invalid syntax"
+    if is_rootless; then
+        run_podman 125 run --device-cgroup-rule="b 7:* rmw" --rm $IMAGE
+        is "$output" "Error: device cgroup rules are not supported in rootless mode or in a user namespace"
```

Comment:
Not a big deal, but if you have to repush again, would you consider just adding `return` here, and removing the long `else` clause? I think that would make the test more intuitively readable, since the `root` case is the only interesting one.

Reply:
sure, pushed with your suggestion